hail_search/hail_search_query.py

```diff
@@ -146,8 +146,32 @@ def import_filtered_table(self, data_type, sample_data, **kwargs):
         if len(family_samples) == 1:
             family_guid, family_sample_data = list(family_samples.items())[0]
             family_ht = hl.read_table(f'{tables_path}/families/{family_guid}.ht')
-            families_ht = self._add_entry_sample_families(family_ht, family_sample_data)
-            families_ht = families_ht.select_globals()
+            families_ht = self._filter_entries_table(family_ht, family_sample_data)
+        else:
+            filtered_project_hts = []
+            exception_messages = set()
+            for project_guid, project_sample_data in project_samples.items():
+                project_ht = hl.read_table(f'{tables_path}/projects/{project_guid}.ht')
+                try:
+                    filtered_project_hts.append(self._filter_entries_table(project_ht, project_sample_data))
+                except HTTPBadRequest as e:
+                    exception_messages.add(e.text)
+
+            if exception_messages:
+                raise HTTPBadRequest(text='; '.join(exception_messages))
+
+            families_ht = filtered_project_hts[0]
+            default_entries = hl.empty_array(families_ht.family_entries.dtype.element_type)
+            for project_ht in filtered_project_hts[1:]:
+                families_ht = families_ht.join(project_ht, how='outer')
+                families_ht = families_ht.select(
+                    filters=families_ht.filters.union(families_ht.filters_1),
+                    family_entries=hl.bind(
+                        lambda a1, a2: a1.extend(a2),
+                        hl.or_else(families_ht.family_entries, default_entries),
+                        hl.or_else(families_ht.family_entries_1, default_entries),
```

Comment:
Where are the `filters_1` and `family_entries_1` fields from?

Reply:
its what happens when you join tables in hail that have the same field names, instead of trying to combine them hail moves them to a new field. Theres no way to avoid it